File: java/src/backjoon/backjoon_2_1.java
package backjoon;

import java.util.Scanner;

public class backjoon_2_1 {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        String input = sc.nextLine();
        String[] splitInput = input.split(" ");
        int[] arr = new int[splitInput.length];

        int count = 0;

        for (String value : splitInput) {
            arr[count++] = Integer.valueOf(value);
        }

        if (arr[0] > arr[1]) {
            System.out.println(">");

        } else if (arr[0] < arr[1]) {
            System.out.println("<");

        } else {
            System.out.println("==");
        }


    }


}

File: 2.kotlin/src/backjoon/backjoon_2_1.kt
package backjoon


fun main(args: Array<String>) {

    var sc: String? = readLine()
    var input: String = sc.toString()

    var splitInput = input.split(" ")
    var arr = Array<Int>(100, { 0 })

    var count: Int = 0
    for (value in splitInput) {
        arr[count++] = value.toInt()
    }

    var A: Int = arr[0]
    var B: Int = arr[1]

    if(A > B) {
        println(">")
    }
    else if( A == B) {
        println("==")
    }
    else {
        println("<")
    }

}
